### PR TITLE
Support Linux ARM64

### PR DESCRIPTION
**Pull request by hrishikesh-k**
### Summary

Attempting to add Linux ARM64 support, similar to: https://github.com/mike-engel/jwt-cli/pull/351/

### Preflight checklist
- [ ] Code formatted rustfmt (`$ cargo fmt`)
- [ ] Code linter check with clippy (`$ cargo clippy`)
- [ ] Relevant tests added
- [ ] Any new documentation added

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -28,6 +28,11 @@ jobs:
             target: x86_64-unknown-linux-musl
             binary_postfix: ""
             label: x86_64 musl
+          - os: ubuntu-latest
+            artifact_prefix: linux-arm64
+            target: aarch64-unknown-linux-gnu
```
Comment (mike-engel) [inline .github/workflows/cd.yml:33]: We might as well add `aarch64-unknown-linux-musl` as well, since this project also provides musl builds for linux

Comment (hrishikesh-k) [reply .github/workflows/cd.yml:33]: Added!

**Comment by mike-engel** [comment]
Hi @hrishikesh-k sorry for the silence. I finally am getting some time here, but noticed this PR was closed. Was that on purpose?

**Comment by hrishikesh-k** [comment]
Sorry, I deleted the repo during my cleanup of old repos. Here's a new PR: https://github.com/mike-engel/jwt-cli/pull/370

**Pull request closed without merging**